### PR TITLE
Narrow down target files by exact match of INPUT_FILES

### PR DESCRIPTION
**Pull request by massongit**
In `entrypoint.sh`, target files are narrowed down by partial match of `INPUT_FILES`.
So if `INPUT_FILES` includes `test/test.txt`, `test/test.txt` and `test/test/test.txt` match.

Therefore, I narrow down target files by exact match of `INPUT_FILES` so that only `test/test.txt` matches.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -107,18 +107,19 @@ if [[ -z "${INPUT_FILES[*]}" ]]; then
...
-  TYPE_CHANGED=$(git diff --diff-filter=T --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
-  UNMERGED=$(git diff --diff-filter=U --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
-  UNKNOWN=$(git diff --diff-filter=X --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
-  ALL_CHANGED_AND_MODIFIED=$(git diff --diff-filter="*ACDMRTUX" --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
-  ALL_CHANGED=$(git diff --diff-filter="ACMR" --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
-  ALL_MODIFIED=$(git diff --diff-filter="ACMRD" --name-only "$PREVIOUS_SHA" "$CURRENT_SHA" | grep -E "(${INPUT_FILES})" | awk -v d="|" '{s=(NR==1?s:s d)$0}END{print s}')
+  INPUT_FILES_PATTERN="^(${INPUT_FILES})"
```
Comment (jackton1) [inline entrypoint.sh:110]: @massongit Thanks for adding this change, 

I'll suggest using [`grep -x`](https://linux.die.net/man/1/grep) for exact matches.

**Review by jackton1 — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (massongit) [reply entrypoint.sh]: I did not know `grep -x`.
I apply your suggestion: https://github.com/tj-actions/changed-files/pull/326/commits/3eca9a027232ddb387e260039c305314020918d5
Thank you!

**Comment by jackton1** [comment]
@all-contributors please add @massongit for code

**Review by jackton1 — COMMENTED** [review]
Looks good. 👍

**Pull request merged**